script/Arcueid/Day 10/10_00_ARC10_5C.txt

```diff
@@ -639,7 +639,7 @@ Even though we're helplessly slaughtered as food...?
 [sha:fc1cd847a33664800a8f8c005f14eb16834edd38]{
 -- Page 548, Offset 25683.
 -- 人間は様々な魔術、神秘学、式典儀礼を基に組織体系を作り、人間以外の霊長類を排除しはじめた。
-Humans created organizations based on various kinds of magecraft, occultism, and ritualism, and then began eliminating non-human Paragons.
+Humans created organizations based on various kinds of magecraft, occultism, and ritualism, and then began eliminating nonhuman Paragons.
```

Comment:
original is correct, non-human is being used as an adjective not noun.

Reply:
you sure i just went with what ngrams said was common :necoarc:
![image](https://user-images.githubusercontent.com/90473809/179354310-df583c76-1ae6-4934-b87a-aad431761ee8.png)

Reply:
both are valid according to [wikipedia](https://en.wikipedia.org/wiki/Non-human), and nonhuman does seem to definitely be more common.